avidemux/common/ADM_render/GUI_simpleRender.cpp

```diff
@@ -73,10 +73,13 @@ bool simpleRender::refresh(void)
 */
 bool simpleRender::displayImage(ADMImage *pic)
 {
-    scaler->convertImage(pic,videoBuffer);
     lock.lock();
+    scaler->convertImage(pic,videoBuffer);
```

Comment:
I think, all changes introduced to avert suspected race condition can be safely reverted now.